000-vip-init.php

```diff
@@ -288,5 +288,29 @@
 	add_filter( 'wp_sitemaps_enabled', '__return_false' );
 }
 
+// We use HyperDB, so DB_* constants are not automatically defined
```

Comment:
```suggestion
// We use HyperDB, so DB_* constants are not automatically defined.
// We define them here in case any customer code is referencing them to avoid fatals errors in PHP 8+.
// As a best practice, these constants should not be used directly (`_doing_it_wrong()`).
```